LogTarget.php

```diff
@@ -165,6 +165,21 @@ protected function collectSummary()
     }
 
     /**
+     * Returns request time in seconds
+     *
+     * @return float request time in seconds.
+     * @see http://php.net/manual/en/function.microtime.php#example-2550
+     * @since 2.0.8
+     */
+    protected function getRequestTime()
```

Comment:
It's the value which is set once at the very beginning of the request. It doesn't change when called multiple times.